plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -44,6 +44,7 @@ import {
   AWSXRayPropagator,
 } from '@opentelemetry/propagator-aws-xray';
 import {
+  SEMATTRS_FAAS_COLDSTART,
```

Comment:
Nit: Would be nice to update the semconv package and use the new constant. Not blocking

Reply:
@dyladan ok, done. just used that new attribute (`faas.coldstart`) from new constant (experimental attribute through incubating)